data/i18n/ja/ja.toml

```
@@ -58,6 +58,9 @@ other = "私は..."
 [docs_label_users]
 other = "ユーザー"
 
+[end_of_life]
+other = "保守終了日"
```

```suggestion
other = "サポート終了日"
```
In current ja contents, we usually don't use "保守" word, instead we use "サポート" on this context, I think :thinking: 
So, is it prefer to use "サポート終了日" instead "保守終了日" as a meaning of `end_of_life` ?

---

I think "サポート終了日" is better than "保守終了日" too

---

Thank you for the quick response!
Could you update with "サポート終了日" ?